<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTransmissionCenter</class>
 <widget class="QWidget" name="DataTransmissionCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>数传中心参数</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>中心名称：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>中心IP地址：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>中心信宿：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>PDXP明文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>PDXP密文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>PDXP明文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>PDXP密文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>FEP明文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>FEP密文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>FEP明文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>FEP密文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>FEP/RECP明文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>FEP/RECP密文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>FEP/RECP明文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>FEP/RECP密文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>描述：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="centerName"/>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="ciphertextSrcPortPDXP"/>
        </item>
        <item row="9" column="1">
         <widget class="QSpinBox" name="plaintextSrcPortFep"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="centerIP"/>
        </item>
        <item row="12" column="1">
         <widget class="QSpinBox" name="ciphertextDestPortFepRECP"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="centerXS"/>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="plaintextDestPortPDXP"/>
        </item>
        <item row="11" column="1">
         <widget class="QSpinBox" name="plaintextDestPortFepRECP"/>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="ciphertextDestPortPDXP"/>
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="plaintextDestPortFep"/>
        </item>
        <item row="10" column="1">
         <widget class="QSpinBox" name="ciphertextSrcPortFep"/>
        </item>
        <item row="8" column="1">
         <widget class="QSpinBox" name="ciphertextDestPortFep"/>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="plaintextSrcPortPDXP"/>
        </item>
        <item row="13" column="1">
         <widget class="QSpinBox" name="plaintextSrcPortFepRECP"/>
        </item>
        <item row="14" column="1">
         <widget class="QSpinBox" name="ciphertextSrcPortFepRECP"/>
        </item>
        <item row="15" column="1">
         <widget class="QTextEdit" name="desc">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editBtn">
       <property name="text">
        <string>编辑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delBtn">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
